<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rFiles.qrc">
    <normaloff>:/Resources/svg/logo.svg</normaloff>:/Resources/svg/logo.svg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="logo">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>788</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="minBtn">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="leftWidget" native="true">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="controlWidget" native="true">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>10</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QPushButton" name="preBtn">
              <property name="text">
               <string></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="playBtn">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string></string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextBtn">
              <property name="text">
               <string></string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="listView">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="rightWidget" native="true">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="hotTab">
            <attribute name="title">
             <string>热门电台</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="promoTab">
            <attribute name="title">
             <string>优选电台</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="moreTab">
            <attribute name="title">
             <string>更多分类</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="statusWidget" native="true">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="timeLabel">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="timeSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="remainingTimeLabel">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="soundBtn">
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="soundSlider">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rFiles.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>minBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
